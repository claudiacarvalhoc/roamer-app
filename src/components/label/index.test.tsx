import React from 'react';
import Label, { LabelProps } from './index';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

const getShallowWrapper = (props) => {
    return shallow(<Label {...props} />);
};

describe('<Label />', () => {
    it('render correctly with highlighted active', () => {
        const props: LabelProps = {
            name: 'test',
            value: 'test',
            highlighted: true,
        };
        const component = getShallowWrapper(props);
        expect(toJson(component)).toMatchSnapshot();
    });
    it('render correctly with highlighted inactive', () => {
        const props: LabelProps = {
            name: 'test',
            value: 'test',
            highlighted: false,
        };
        const component = getShallowWrapper(props);
        expect(toJson(component)).toMatchSnapshot();
    });
});
