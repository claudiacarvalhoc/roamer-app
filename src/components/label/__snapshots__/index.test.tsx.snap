// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Label /> render correctly with highlighted active 1`] = `
<div
  className="label"
>
  <p
    className="name"
  >
    test
  </p>
  <p
    className="value highlighted"
  >
    test
  </p>
</div>
`;

exports[`<Label /> render correctly with highlighted inactive 1`] = `
<div
  className="label"
>
  <p
    className="name"
  >
    test
  </p>
  <p
    className="value"
  >
    test
  </p>
</div>
`;
